### DISCUSSION TITLE
Pairing hackathon with community members

### DISCUSSION THREAD

**Discussion by jespino** [mattermost/community#128] [General Ideas]
Make a 2 days hackathon of maybe 3 or 4 hours per day, where core developers of mattermost pair with community members (of any level of experience) and help them to contribute to mattermost. That can start (when needed) with a walkthrough in the code, and then help them to understand what is expected and what are the problems. Also trying to accelerate reviewes from other members of Mattermost to get the changes in ASAP to make the experince super rewarding.

**Comment by chetanyakan** [comment]
OR incentivize MM staff members to pair with the community members more.
Like while doing a code review.

**Comment by esethna** [comment]
Love this idea. "Paired" hackathos also allow non-code people to participate, like PM and Design!
